test/testdata/infer/generics/all_bottom.rb

```diff
@@ -0,0 +1,30 @@
+# typed: true
+
+class Base1
+  extend T::Generic
+  Elem = type_member
+end
+
+class Base2
+  extend T::Generic
+  Elem = type_member
+end
+class Child < Base2
```

Comment:
Can we call this `Child2` ? when i was reading the test first i thought that `Child < Base1`